Assets/Scripts/Layers/Adapters/DataTypeAdapters/WFSGeoJSONImportAdapter.cs

```diff
@@ -206,6 +217,18 @@ public bool HasGetFeatureAsGeoJSON()
                 return true;
             }
 
+            public string GetGeoJsonOutputFormatString()
+            {
+                XmlNode getFeatureOperationNode = ReadGetFeatureNode(this.xmlDocument, this.namespaceManager);
+                if (getFeatureOperationNode == null)
+                {
+                    Debug.Log("<color=orange>WFS GetFeature operation not found.</color>");
```

Comment:
is dit nog nodig? als het een error is misschien LogError van maken?

Reply:
Totdat we fatsoenlijke errorafhandeling hebben wel helaas; een van de pijnpunten is dat we niet iets terugkoppelen aan de gebruiker of onszelf. Ik vraag me zelfs af of we niet een in-app console moeten hebben oid